controller/AuthController.ts

```diff
@@ -0,0 +1,41 @@
+import { Request, Response, Application } from 'express';
+import { Login } from '../model/Auth';
+import AuthService from '../service/AuthService';
+import axios from 'axios';
+import { Job } from '../model/Job';
+
+const authService: AuthService = new AuthService();
+
+module.exports = function (app: Application) {
+  app.get('/login', async (req: Request, res: Response) => {
+    res.render('login');
+  });
+
+  app.post('/login', async (req: Request, res: Response) => {
+    const data: Login = req.body;
+    
+    try {
+      req.session.token = await authService.login(data);
+      const response = await axios.get('/jobs');
+      const jobs: Job[] = response.data;
+      console.log(req.session.token);
+      res.render('view-all-jobs', { token: req.session.token, jobs });
```

Comment:
you shouldn't do this here. just redirect to the jobs endpoint

Reply:
redirected